### ISSUE TITLE
Options based on type (Error, Success, Info, Warning)

### ISSUE THREAD

**Issue by RhysyG** [CodeSeven/toastr#417]
Hey!

It would be great to have options by type (Error, Success, Info, Warning). For example, 10 secs with a close for error messages, 5 secs with no close on positive feedback ect..

I can do this now but it's all manual overrides and it would be great to change a type in one hit.

Thanks!

**Comment by NaomiN** [comment]
Right. I need to be able to not dismiss errors toasts by clicking on them and also have Copy Text and Send Email for the errors. Do you think it's possible to add?

Thanks in advance.

**Comment by akdsco** [comment]
Yes, copy text feature would be amazing, however, it looks like this repo isn't being updated whatsoever.. :cry:
